.gitignore

```diff
@@ -35,4 +36,4 @@ docs
 package-lock\.json
 
 # Visual Studio Code
-.vscode/
+# .vscode/
```

Comment:
if you think it's useful, you can remove it

Reply:
Yep. I think that vast majority of JS devs use vscode so it's useful to have a debugger config.

Action: Removed the `vscode/` from gitignore